modules/consensus/synchronize_test.go

```diff
@@ -108,7 +107,6 @@ func TestBlockHistory(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow()
 	}
-	t.Parallel()
```

Comment:
`TestSynchronize` has been timing out for me today when run in parallel (running the test on its own is fine).  The test times out on line 84 while waiting for `cst1.cs.dbBlockHeight()` to catch up to `cst2.cs.dbBlockHeight()` after reconnecting the gateways.